DQM/L1TMonitor/python/L1TEmulatorMonitor_cff.py

```diff
@@ -41,15 +41,15 @@
 from Configuration.ProcessModifiers.dqmPerLSsaving_cff import dqmPerLSsaving
 dqmPerLSsaving.toModify(l1TdeRCT, perLSsaving=True)
 
-l1TdeRCTRun1 = l1TdeRCT.clone()
-l1TdeRCT.rctSourceData = 'caloStage1Digis'
-#l1TdeRCT.gctSourceData = 'caloStage1Digis'
-l1TdeRCT.rctSourceEmul = 'valRctDigis'
-
-l1TdeRCTfromRCT = l1TdeRCT.clone()
-l1TdeRCTfromRCT.rctSourceData = 'rctDigis'
-l1TdeRCTfromRCT.HistFolder = cms.untracked.string('L1TEMU/L1TdeRCT_FromRCT')
-
+l1TdeRCTRun1 = l1TdeRCT.clone(
+rctSourceData = 'caloStage1Digis',
```

Comment:
this change is not the same: l1TdeRCTRun1  should be a frs clone of l1TdeRCT and then the pvalues of l1TdeRCT should be changed, please leave as it was

Reply:
Okay, Thanks. Updated as it was in the new commit.